.readthedocs.yaml

```diff
@@ -11,6 +11,6 @@ sphinx:
 
 # Optionally set the version of Python and requirements required to build your docs
 python:
-   version: 3.7
+   version: 3.11
```

Comment:
I think you need to set `os: ubuntu- 22.04`
https://docs.readthedocs.io/page/config-file/v2.html

Reply:
So maybe the file should look more like this:

```
build:
  os: ubuntu-22.04
  tools:
    python: "3.11"


python:
   install:
   - requirements: docs_requirements.txt
   
```

Reply:
Thanks!